src/main/java/com/google/cloud/tools/managedcloudsdk/install/SdkInstaller.java

```diff
@@ -52,6 +55,20 @@
     this.installerFactory = installerFactory;
```

Comment:
This constructor can just call `this(...)` to delegate to the new constructor? (I'm not sure what the best practice is.)

Reply:
Definitely, good call. Updated.